samples/CommunityToolkit.Maui.Sample/Platforms/Android/AndroidManifest.xml

```diff
@@ -10,6 +10,7 @@
   <uses-permission android:name="com.sec.android.provider.badge.permission.WRITE" />
 
 	<uses-permission android:name="android.permission.RECORD_AUDIO" />
+	<uses-permission android:name="android.permission.CAMERA" />
```

Comment:
Reminder that we'll need to add this to the `Platform specific initialization` section of the `CameraView` docs